src/Microsoft.DotNet.XliffTasks/Microsoft.DotNet.XliffTasks.csproj

```
@@ -1,11 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetToolCurrent);$(NetFrameworkToolCurrent)</TargetFrameworks>
+    <!--
+       We need to target net8.0 while Arcade SDK 8 is being used.
+       Xliff tasks were moved from dotnet/xliff-tasks to Arcade in V9, so they are not available in release/8.0 branch.
+    -->
+    <TargetFrameworks>$(NetToolMinimum);$(NetFrameworkToolCurrent)</TargetFrameworks>
     <IsPackable>true</IsPackable>
     <RootNamespace>XliffTasks</RootNamespace>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <PackageReadmeFile>README.md</PackageReadmeFile>
+
+    <!-- The subdirectory name is the value of $(MSBuildRuntimeType). -->
+    <BuildTaskTargetTfmSpecificFolder>tools/Core</BuildTaskTargetTfmSpecificFolder>
+    <BuildTaskTargetTfmSpecificFolder Condition="'$(TargetFramework)' == '$(NetFrameworkToolCurrent)'">tools/Full</BuildTaskTargetTfmSpecificFolder>
```

Use a condition in L15 to avoid the property redefine when compiling against .NET Framework. Also use better framework names:

```suggestion
    <BuildTaskTargetTfmSpecificFolder Condition="'$(TargetFramework)' == '$(NetToolMinimum)'">tools/net</BuildTaskTargetTfmSpecificFolder>
    <BuildTaskTargetTfmSpecificFolder Condition="'$(TargetFramework)' == '$(NetFrameworkToolCurrent)'">tools/netframework</BuildTaskTargetTfmSpecificFolder>
```